### PR TITLE
Build Python Module on Appveyor

### PR DESCRIPTION
**Pull request by yannlabou**

### CODE REVIEW

**Comment by yannlabou** [comment]
@ihhub Please review :)

**Comment by ihhub** [comment]
It looks like it's built. I think the next step is to copy built file into examples/python/ as well with image from examples/_image and run bitmap_operation.py file. What do you think?

File: appveyor.yml [hunk 1]
```diff
@@ -12,7 +12,26 @@ version: 1.0.{build}
...
+    - set PATH=C:\Python37;%PATH%
+    - msbuild /version
+    - python --version
+  install:
+    - if not exist C:\ProgramData\chocolatey\bin\swig.exe choco install swig --version 3.0.12 --yes --limit-output
+  build_script:
+    - cmd: cd C:\projects\penguinv\src\python
+    - cmd: python setup.py build_ext --inplace
+  test_script:
+    - cmd: python bitmap_operation.py
```
Comment (ihhub) [inline appveyor.yml]: We need to execute this from examples/python directory.

**Comment by yannlabou** [comment]
Thanks for the last changes.

**Comment by ihhub** [comment]
@yannlabou you're welcome!
Anyway, I don't see any output from python file as it should print something. Have you tried to run this python script on your machine without image copying?

**Comment by ihhub** [comment]
Ah, sorry, it's my mistake. We read the image from proper location.

**Comment by ihhub** [comment]
@yannlabou thank a lot for fixing this issue!

**Pull request merged**